modules/permutiveRtdProvider.js

```diff
@@ -222,10 +242,19 @@ function getCustomBidderFn (moduleConfig, bidder) {
  * @return {Object} Bidder function
  */
 function getDefaultBidderFn (bidder) {
+  const isPStandardTargetingEnabled = (data, acEnabled) => {
+    return (acEnabled && data.ac && data.ac.length) || (data.ssp && data.ssp.cohorts.length)
+  }
+  const pStandardTargeting = (data, acEnabled) => {
+    const ac = (acEnabled) ? (data.ac ?? []) : []
+    const ssp = data?.ssp?.cohorts ?? []
+    return [...new Set([...ac, ...ssp])]
+  }
   const bidderMap = {
     appnexus: function (bid, data, acEnabled) {
-      if (acEnabled && data.ac && data.ac.length) {
-        deepSetValue(bid, 'params.keywords.p_standard', data.ac)
+      if (isPStandardTargetingEnabled(data, acEnabled)) {
+        const segments = pStandardTargeting(data, acEnabled)
+        deepSetValue(bid, 'params.keywords.p_standard', segments)
```

Comment:
you guys should consider yourself on notice to remove these special handlings, move them into their respective bid adapters, expect someone else moves them for you or simply deletes them, or expect not to be in the next version of prebid

Reply:
Hi Patrick, thanks for the feedback. I'm already working on removing the appnexus special handling, I just wanted to do it as a separate PR in order to simplify reviewing.

For the other SSPs for which we have special handling - we're looking into them. I don't have the full context myself, but I understand it's not as simple as migrating to the `ortb2.user.data` field. I'll catch up internally with my team to ensure we don't break any existing targeting, as this might impact our customer's revenue!

Regardless, thanks for the heads-up that this method will stop working soon - we'll prioritise updating this to avoid any issues.